lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -416,6 +416,13 @@ describe_instances(module.client('ec2'), InstanceIds=['i-123456789'])
 The call will be retried the specified number of times, so the calling functions
 don't need to be wrapped in the backoff decorator.
 
+You can also use customization for `retries`, `delay` and `max_delay` parameters used by
+`AWSRetry.jittered_backoff` API using module params. You can take a look into
+[cloudformation](/lib/ansible/modules/cloud/amazon/cloudformation.py) module for example.
+
+To make all Amazon modules uniform, please use module param as `backoff_retries` for `retries`,
+`backoff_delay` of `delay` and `backoff_max_delay` for `max_delay`.
```

Comment:
of->for, or just change the whole thing to explain that the backoff_ prefix should be added, and list the three examples:

> To make all Amazon modules uniform, prefix the module param with `backoff_`, so `retries` becomes `backoff_retries` and likewise with `backoff_delay` and `backoff_max_delay`.

Reply:
Done.